server/service.go

```diff
@@ -242,12 +250,18 @@ func (m *BoostService) handleGetHeader(w http.ResponseWriter, req *http.Request)
 	vars := mux.Vars(req)
 	slot := vars["slot"]
 	parentHashHex := vars["parent_hash"]
-	pubkey := vars["pubkey"]
+
+	publicKey, err := types.HexToPubkey(vars["pubkey"])
+	if err != nil || len(vars["pubkey"]) != 98 {
```

Comment:
```suggestion
	if err != nil {
```

size is already checked in HexToPubkey

Reply:
My bad, I forgot to create an issue about this but I added this check to prevent `TestGetHeader/Invalid pubkey length` to fail.
If you remove this condition you'll see this test fail.

Reply:
I've [opened a PR](https://github.com/flashbots/go-boost-utils/pull/26) which fixes the issue in go-boost-utils.

Reply:
Fixed in db51af85f88f43ca0bb9398c78abfbc409fc67cc.